core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1241,6 +1241,45 @@ public String toString() {
           .setScope(Scope.MASTER)
           .build();
 
+  //
+  // Mount table journal related properties
```

Comment:
I don't these parameters are necessary. The journal is a not actually a mount point. There is already a config prefix for the journal config: https://github.com/Alluxio/alluxio/blob/v2.4.1/core/common/src/main/java/alluxio/conf/PropertyKey.java#L1163

Reply:
Thanks for remind me, I've enlarge the access scope of the `getJournalUfsConf ` from `protected` to `public` and turn to use `getJournalUfsConf ` instead.